### PR TITLE
new change to send from my fork

### PR DESCRIPTION
**Pull request by AdrianaRs13**

### CODE REVIEW

**Review by ihernandez-ias — CHANGES_REQUESTED** [review]
I think that this is wrong

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Promises fn utils
+
 This package is intended to provide a wide a variety of decorator functions to easily add capabilities to your functions which returns promises and this without modify the function's interface.
 
+#Holaaaaaaaaaaaaaaaa
```
Comment (ihernandez-ias) [inline README.md:5]: Que significa "Hola" I don't understand

Comment (AdrianaRs13) [reply README.md:5]: Solved

**Review by ihernandez-ias — APPROVED** [review]
LGTM

**Review by ihernandez-ias — APPROVED** [review]
LGTM

**Review by isaachq — APPROVED** [review]

**Review by isaachq — COMMENTED** [review]
test

**Pull request closed without merging**